### PR TITLE
6.0

### PR DESCRIPTION
**Pull request by hoskinsj**

### CODE REVIEW

File: seeker/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-__version__ = '6.2.1'
+__version__ = '6.3.0'
```
Comment (tomlinsj) [inline seeker/__init__.py:1]: This seems more like a 6.2.2 to me. we didn't add functionality here just removed some depreciated code

Comment (hoskinsj) [reply seeker/__init__.py:1]: Updated, moved it to 6.2.3, since 6.2.2 was recently released

Comment (tomlinsj) [reply seeker/__init__.py:1]: having slept on it... I think 6.3.0 make sense we did add functionality, support for Django 2.x

Comment (hoskinsj) [reply seeker/__init__.py:1]: Updated

**Pull request merged**